pkg/sdkserver/sdkserver.go

```diff
@@ -457,7 +458,12 @@ func (s *SDKServer) patchGameServer(ctx context.Context, gs, gsCopy *agonesv1.Ga
 		return nil, err
```

Comment:
Do we also need to account for this error return as well?

Reply:
Just double checked - this is already wrapped, so we don't have to re-wrap it.

Also, if there is an issue with creating a patch, that's really shouldn't happen and should definitely be logged.